third_party/blink/renderer/modules/media_controls/media_controls_impl.cc

```diff
@@ -176,6 +176,9 @@ bool ShouldShowPictureInPictureButton(HTMLMediaElement& media_element) {
 }
 
 bool ShouldShowCastButton(HTMLMediaElement& media_element) {
+#if BUILDFLAG(IS_COBALT)
+  return false;
```

Comment:
Would it be possible to do e.g.
```
  HTMLMediaElementRemotePlayback::SetBooleanAttribute(
      *element, html_names::kDisableremoteplaybackAttr, true);
```
in a central place, e.g. `MediaControlsImpl::Create()`? That
way JS could see the `disableRemotePlayback` attribute set.